chaste_codegen/_jacobian.py

```diff
@@ -0,0 +1,33 @@
+import sympy as sp
+
+
+def get_jacobian(state_vars, derivative_equations):
```

Comment:
Would be good to add to the docstrings for both functions things like what is returned, what the parameters are.

Reply:
Yeah fair point I'll do that.